health/micro-ui-react19/web/builds/README.md

```
@@ -43,90 +43,115 @@ DIGIT-Frontend/
 
 This application supports three distinct build variants:
 
-### 🏥 **Console** (`/console/`)
+### **Console** (`/console/`)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)